tests/test_client.py

```diff
@@ -1493,6 +1493,131 @@ def test_deleting_skip_bugsnag_attr_allows_notify(self):
 
         assert self.sent_report_count == 1
 
+    @pytest.mark.skipif(
+        sys.version_info < (3, 11),
+        reason="requires python 3.11 or higher"
+    )
+    def test_exception_groups_are_unwrapped(self):
+        # disable send_code so we can assert against stacktraces more easily
+        self.client.configuration.configure(send_code=False)
+
+        self.client.notify(fixtures.exception_group_with_no_cause)
+
+        assert self.sent_report_count == 1
+
+        payload = self.server.received[0]['json_body']
+        exceptions = payload['events'][0]['exceptions']
+
+        # there should be 1 ExceptionGroup with 4 sub-exceptions
+        assert len(exceptions) == 5
+
+        assert exceptions[0] == {
+            'message': 'the message of the group (4 sub-exceptions)',
+            'errorClass': 'builtins.ExceptionGroup',
```

Comment:
Our `fully_qualified_class_name` is appending `builtins.` here, which is not technically _wrong_ but is unexpected

Fixing this is trivial but I've split this into a separate task as:

1. I'm not sure why this happens for `ExceptionGroup` but not other built-in exceptions
2. `fully_qualified_class_name` is also used when applying `ignore_classes`, so we need to be careful not to change its behaviour in a way that would break ignoring errors